local-tests/chainrunner/chain.py

```diff
@@ -21,32 +22,48 @@ def __init__(self, workdir):
         os.makedirs(workdir, exist_ok=True)
         self.path = op.abspath(workdir)
         self.nodes = []
+        self.validator_nodes = []
+        self.nonvalidator_nodes = []
 
     def __getitem__(self, i):
         return self.nodes[i]
 
     def __iter__(self):
         return iter(self.nodes)
 
-    def bootstrap(self, binary, accounts, **kwargs):
-        """Bootstrap the chain. `accounts` should be a list of strings.
+    def bootstrap(self, binary, validator_accounts, accounts=None, **kwargs):
```

Comment:
Maybe naming these arguments `validators` and `nonvalidators` would be more descriptive?

Reply:
done